### DISCUSSION TITLE
collegamento tra home assistant e putty

### DISCUSSION THREAD

**Discussion by dj8simo6** [indomus/forum#2090] [HUB: Home Assistant]
Salve ragazzi buonasera ,ieri ho provato a migrare il mio sistema dalla sd al ssd . Ho seguito per filo e per segno la vostra guida.... Ho ripritinato il backup ed e tutto funzionante .Quando provo a collegarmi  tramite putty e inserisco username e password , mi da password sbagliata . Le ho provare tutte ma non mi fa accedere , come mai? potete aiutarmi ? grazie

**Comment by NdR91** [comment]
Eh, ma noi come possiamo sapere quale sia la tua password...

**Reply by dj8simo6** [reply]
ma io la password come l'username le so ! non capisco perche mi da access danied .. non le ho mai modificate

**Reply by Nik71git** [reply]
permission? vai con sudo chmod (777 meglio)

**Reply by NdR91** [reply]
> ma io la password come l'username le so ! non capisco perche mi da access danied .. non le ho mai modificate

Ma sei su RaspOS o HAOS?
L'errore esatto qual'è? `access denied` non significa necessariamente che sia un errore di password

> permission? vai con sudo chmod (777 meglio)

Cosa????

**Reply by Nik71git** [reply]
@NdR91 dicevo solo se aveva un problema di permission... per verificare dai accesso a tutto e a tutti, ripeto solo per prova.... Ma leggendo oltre vedo che i problemi non sono questi.

**Reply by dj8simo6** [reply]
si mi da access danied . Home assistan OS 9.3

**Reply by snakuzzo** [reply]
L'accesso in ssh verso ha os va abilitato tramite addon (se non ricordo male). 
Ssh & web terminal ad esempio. Lì segui la doc dell'addon e c'è scritto come fare per impostare le utenze

**Reply by NdR91** [reply]
Esattamente, hai già scaricato ed avviato addon SSH?

**Reply by dj8simo6** [reply]
si e stato ripristinato dal backup ed e funzionante

**Reply by snakuzzo** [reply]
Allora controlla i log dell'addon...visto che è quello che ti permette di collegarti in ssh

**Reply by dj8simo6** [reply]
per sicurezza ho disinstallato e re installato riesco solo ad entrare come se fosse da terminale

**Reply by snakuzzo** [reply]
???

**Reply by dj8simo6** [reply]
e come se aprissi il terminale e non ssh

**Reply by snakuzzo** [reply]
Ma cosa vuol dire "è come se aprissi" ?

**Reply by snakuzzo** [reply]
Ti colleghi da remoto? Sì o no?
Hai letto la doc dell'addon? Sì o no?
Hai visto i log dell'addon? Sì o no?

**Reply by dj8simo6** [reply]
magari non mi spiego bene io! la sessione da terminale mi funziona benissimo ho controllato anche i log ... il problema non e questo ! a me serve accedere ad ssh tramite putty .Prima inserivo pi come utente e passw raspberry ed ero dentro ....adesso mi da access danied , ma io non ho modificato nulla !

**Reply by snakuzzo** [reply]
Ma io prima che ti ho scritto?
Hai letto?
Premesso che non so tu cos'abbia fatto in precedenza, ma per l'accesso in ssh devi configurare l'addon specificando lì user e password da usare

**Reply by snakuzzo** [reply]
Se poi pretendi che uno ti dia una risposta immaginando o guardando in una sfera magica per capire cos'hai fatto sulla vecchia installazione, allora non è qui che devi chiedere :)

**Reply by snakuzzo** [reply]
Quando si scrive "prima funzionava, ora no"...vuol dire che in precedenza hai fatto qualcosa che magari non hai afferrato bene ed ora non sai come uscirne

**Reply by dj8simo6** [reply]
ripeto ho seguito la vostra guida per migrare il sistema su ssd .... ho ripristina il backup ed e tutto funzionante .... riesco ad accedere solo al terminale e non ad ssh ... anche inserendo username e password del componente aggiuntivo , mi colelga al terminal e non ad ssh

**Reply by snakuzzo** [reply]
Ma scusami un attimo...
spiegami "tu" cosa intendi per "terminal" e cosa intendi per "ssh"

**Reply by snakuzzo** [reply]
Vediamo se ho capito cosa intendi...
Stai forse cercando di dirmi che per te un comportamento come nello screenshot che segue è sbagliato ? (a sx la webgui di ha os con web terminal e a dx una connessione in ssh verso ha os)

![image](https://user-images.githubusercontent.com/7205371/202727219-d73033fb-0f7a-417a-beff-247349c20d67.png)

Perché se per te questo è sbagliato...ti dico subito che ti stai incartando da solo, perché è giusto che sia così :)

Fammi sapere

**Reply by dj8simo6** [reply]
non intendevo questo ! lo so che sono uguali e normale ... io sia da terminal nel menu a sinistra, che da putty mi si aprono queste due pagine che hai messo ... Ora ho  tolto l ssd e rimontato la  sd originale ed ho capito dove sta il problema . Nella scheda sd gira debian linux 11 mentre l installazione fatta nella ssd monta home assistant os

**Reply by snakuzzo** [reply]
...senza parole :D
